build.sh

```diff
@@ -17,6 +17,7 @@ BUILD_DIR=${BUILD_DIR:-${UFS_MODEL_DIR}/build}
 mkdir -p ${BUILD_DIR}
 
 [[ -n "${CCPP_SUITES:-""}" ]] && CMAKE_FLAGS+=" -DCCPP_SUITES=${CCPP_SUITES}"
+[[ -n "${MAPL_ROOT:-""}" ]] && CMAKE_FLAGS+=" -DCMAKE_MODULE_PATH=${MAPL_ROOT}/share/MAPL/cmake"
```

Comment:
This location (`${MAPL_ROOT}/share/MAPL/cmake`) should be added to `CMAKE_MODULE_PATH` somewhere in some CMakeLists.txt in gocart code so that people can build the code without running this build script i.e. by just running cmake (and of course loading required modules where necessary).

Reply:
This would affect NASA's build process.

Reply:
Adding `-DCMAKE_MODULE_PATH` arguments to `build.sh` is not desirable.
One should be able to build without using `build.sh` with options that the model requires; e.g. APP, CCPP_SUITES.
Handling package dependencies should be done by the package providers.

See the comment below on why this is essentially is bugfix for MAPL.